docs/scripts/degen.rst

```diff
@@ -0,0 +1,67 @@
+degen
+=====
+Find genes where a sequence has degenerate bases.
+
+How-to
+=====
+Usage:
```

Comment:
I think we might want to keep usage statements out of the docs as they tend to change and then get out of sync.

I think it is best to just ensure users can use the ``--help`` to get it if they want